.gitignore

```diff
@@ -2,3 +2,4 @@ config.env
 node_modules/
 !test/unit/fixtures/**/node_modules
 tmp
+**/__preferences.json
```

Comment:
This file should be created in HOME/.tessel/ where the rest of the user-owned stuff is